test/integration/react-18/test/index.test.js

```diff
@@ -67,11 +68,13 @@ function runTestsAgainstRuntime(runtime) {
           invalidPage.write(`export const value = 1`)
         }
         nextConfig.replace("// runtime: 'edge'", `runtime: '${runtime}'`)
+        indexPage.replace("// runtime: 'edge'", `runtime: '${runtime}'`)
```

Comment:
Setting page runtime to edge, this wasn't coverred before cause it will be statically optimized.
Adding this to avoid the basic build failed when removing the nodejs polyfills